holoviews/plotting/bokeh/callbacks.py

```diff
@@ -249,8 +303,12 @@ class RangeXYCallback(Callback):
     handles = ['x_range', 'y_range']
 
     def _process_msg(self, msg):
-        return {'x_range': (msg['x0'], msg['x1']),
-                'y_range': (msg['y0'], msg['y1'])}
+        data = {}
+        if 'x0' in msg and 'x1' in msg:
+            data['x_range'] = (msg['x0'], msg['x1'])
+        if 'y0' in msg and 'y1' in msg:
+            data['y_range'] = (msg['y0'], msg['y1'])
+        return data
 
 
 class RangeXCallback(Callback):
```

Comment:
``RangeXY`` is really the union of ``RangeX`` and ``RangeY``. Me might want to consider generalizing this idea of a union so we can build things like ``RangeXY`` out of the component pieces.